content/writing_tests/_index.md

```diff
@@ -65,7 +65,7 @@ Due to those reasons, we would advise you not to use the `.then(...)` syntax, un
 
 #### Async Functions
 
-Most functions which implement the <a href="https://sahipro.com/docs/sahi-apis/index.html" target="_blank">Sahi DSL</a> - recognizable by a prefixed underscore - return a Promise. That means, that you should put an `await` in front of it. Exceptions from this pattern are [Accessor functions](/apidoc/sakuli-legacy/interfaces/accessorapi.html), which create query objects to access elements in the DOM. You can check if the function returns a Promise in the [API docs of SahiAPI](apidoc/sakuli-legacy/interfaces/sahiapi.html). If you are not sure that you need an `await`, you can put it anyway since the function will be executed as expected. Considering that this is not a good practice, we are working on tools which will help you identify async functions like <a href="https://github.com/sakuli/sakuli/tree/feature/ts-support" target="_blank">Typescript support</a>.
+Most functions which implement the <a href="https://sahipro.com/docs/sahi-apis/index.html" target="_blank">Sahi DSL</a> - recognizable by a prefixed underscore - return a Promise. That means, that you should put an `await` in front of it. Exceptions from this pattern are [Accessor functions](/apidoc/sakuli-legacy/interfaces/accessorapi.html), which create query objects to access elements in the DOM. You can check if the function returns a Promise in the [API docs of SahiAPI](/apidoc/sakuli-legacy/interfaces/sahiapi.html). If you are not sure that you need an `await`, you can put it anyway since the function will be executed as expected. Considering that this is not a good practice, we are working on tools which will help you identify async functions like <a href="https://github.com/sakuli/sakuli/tree/feature/ts-support" target="_blank">Typescript support</a>.
```

Comment:
```suggestion
Most functions which implement the <a href="https://sahipro.com/docs/sahi-apis/index.html" target="_blank">Sahi DSL</a> - recognizable by a prefixed underscore - return a Promise. That means, that you should put an `await` in front of it. Exceptions from this pattern are [Accessor functions](/apidoc/sakuli-legacy/interfaces/accessorapi.html), which create query objects to access elements in the DOM. You can check if the function returns a Promise in the [API docs of SahiAPI](/apidoc/sakuli-legacy/interfaces/sahiapi.html). If you are not sure that you need an `await`, you can put it anyway since the function will be executed as expected. Considering that this is not a good practice, we are working on tools which will help you identify async functions like <a href="https://github.com/sakuli/sakuli/tree/feature/ts-support" target="_blank" rel="noopener">Typescript support</a>.
```

https://www.christoph-freyer.at/blog/links-absichern-mit-rel-noopener/#.Xg22MxdKiL4